### PR TITLE
add lookup_item()

### PR DESCRIPTION
**Pull request by adir-lb**
This function returns the item with the given name under the given
parent item.

### CODE REVIEW

File: src/procstat.h [hunk 1]
```diff
@@ -323,4 +323,7 @@ int procstat_create_histogram_u32_series(struct procstat_context *context, struc
 
 void procstat_histogram_u32_add_point(struct procstat_histogram_u32 *series, uint32_t value);
 
+struct procstat_item *procstat_lookup_item(struct procstat_context *context,
```
Comment (sashas-lb) [inline src/procstat.h]: can you please move this defenition "up" where rest of generic defenitions are placed

Comment (adir-lb) [reply src/procstat.h]: no problem, but according to this logic also `procstat_remove*()` should be moved, right?

Comment (adir-lb) [reply src/procstat.h]: done

**Comment by adir-lb** [comment]
@sashas-lb fixed the comment. do i have +1?

**Comment by sashas-lb** [comment]
can you please add some stupid test for lookup and in any case you have +1

**Comment by adir-lb** [comment]
@sashas-lb done

**Pull request merged**